sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -0,0 +1,197 @@
+
+### Training models
+Using the `CustomFormClient`, you can train a machine-learned model on your own form type.  The resulting model will be able to extract values from the types of forms it was trained on.
+
+#### Training without labels
+A model trained without labels uses unsupervised learning to understand the layout and relationships between field names and values in your forms. The learning algorithm clusters the training forms by type and learns what fields and tables are present in each form type. 
+
+This approach doesn't require manual data labeling or intensive coding and maintenance, and we recommend you try this method first when training custom models.
+
+#### Training with labels
+A model trained with labels uses supervised learning to extract values you specify by adding labels to your training forms.  The learning algorithm uses a label file you provide to learn what fields are found at various locations in the form, and learns to extract just those values.
+
+This approach can result in better-performing models, and those models can work with more complex form structures.
+
+### Extracting values from forms
+Using the `CustomFormClient`, you can use your own trained models to extract field values and locations, as well as table data, from forms of the type you trained your models on.  The output of models trained with and without labels differs as described below.
+
+#### Using models trained without labels
+Models trained without labels consider each form page to be a different form type.  For example, if you train your model on 3-page forms, it will learn that these are three different types of forms.  When you send a form to it for analysis, it will return a collection of three pages, where each page contains the field names, values, and locations, as well as table data, found on that page.
+
+#### Using models trained with labels
+Models trained with labels consider a form as a single unit.  For example, if you train your model on 3-page forms with labels, it will learn to extract field values from the locations you've labeled across all pages in the form.  If you sent a document containing two forms to it for analysis, it would return a collection of two forms, where each form contains the field names, values, and locations, as well as table data, found in that form.  Fields and tables have page numbers to identify the pages where they were found.
+
+### Managing Custom Models
+Using the `CustomFormClient`, you can get, list, and delete the custom models you've trained.  You can also view the count of models you've trained and the maximum number of models your subscription will allow you to store.
+
+## Examples
+The following section provides several code snippets illustrating common patterns used in the Form Recognizer .NET API.
+
+### Extracting receipt values with a long-running operation
+```C#
```

Comment:
Snippets?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10484